retriever/lib/defaults.py

```diff
@@ -80,3 +83,11 @@
   doi={10.1371/journal.pone.0065848}
 }
 """
+try:
```

Comment:
We should try to keep import statements at the top unless there is a good reason to keep them here